release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -48,9 +48,136 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### Strand dump showing the total strand group and strand count in a Ballerina program
+
+The total number of strand groups and strands created in a Ballerina program is now shown in the strand dump.
+
+#### Allowing ambiguous target types in `fromJsonWithType` & `cloneWithType`
+
+When the user-specified target type was a union type consisting of multiple convertible types for a given source value, earlier `fromJsonWithType` & `cloneWithType` returned an error. Now for such user-specified target types, the operation succeeds without returning an error, returning a value whose inherent type is as follows. If the given source value belongs to at least one of the member types of the target union type, the inherent type of the returned value is the first found type to which the source value belongs. Otherwise, the inherent type of the returned value is the first convertible type in the union.
```

Comment:
```suggestion
The `fromJsonWithType` & `cloneWithType` types returned an error in the earlier implementation when the user-specified target type was a union type consisting of multiple convertible types for a given source value. Now, for such user-specified target types, the operation succeeds without returning an error by returning a value whose inherent type is as follows. If the given source value belongs to at least one of the member types of the target union type, the inherent type of the returned value is the first found type to which the source value belongs. Otherwise, the inherent type of the returned value is the first convertible type in the union.
```